### PR TITLE
ansible-trackster standardization

### PR DESCRIPTION
**Pull request by drosofff**
This pr is an attempt to standardize and refresh the ansible-trackster role.

- Use the more common galaxy_server_dir variable
- drop comments on cloud
- fetch directly a compiled version of bedtools (no need to run make)
- provide a local set of refreshed .len files in the repo.
- targets the root-galaxy/.venv/bin directory for the executables instead of /usr/local/bin

Galaxykickstart was used to test this branch ans is going to use it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,24 +3,19 @@ This role is used to setup [Trackster][trackster] genome browser in Galaxy.
 Requirements
 ------------
 The role has been developed and tested on Ubuntu 14.04 and 16.04. It uses
```
Comment (afgane) [inline README.md:5]: I think we can skip 14.04 at this point, and add 18.04 while at it.

File: README.md [hunk 2]
```diff
@@ -3,24 +3,19 @@ This role is used to setup [Trackster][trackster] genome browser in Galaxy.
...
 Variables
 ---------
-If using this role in the context of [Galaxy CloudMan][cm], note that some of these
-variables need to match equally named ones from the [`ansible-cloudman`][acm]
-role.
 
  - `galaxy_manage_trackster`: (default: `yes`) if set, the role will run
  - `galaxy_user_name`: (default: `galaxy`) system username used for Galaxy
- - `galaxyFS_base_dir`: (default: `/mnt/galaxy`) the base path under which the
+ - `galaxy_server_dir`: (default: `/home/galaxy/galaxy`) the base path under which the
```
Comment (afgane) [inline README.md:13]: Shouldn't this be just `/home/galaxy`? Or even something like `/galaxy/server`?

File: README.md [hunk 3]
```diff
@@ -3,24 +3,19 @@ This role is used to setup [Trackster][trackster] genome browser in Galaxy.
...
     galaxy file system is planned to be placed
- - `len_file_path`: (default: `"{{ galaxyFS_base_dir }}/galaxy-app/config/len"`)
-    the default location where Trackster's `.len` files should be placed
- - `bedtools_version`: (default: `2.25.0`) the version of BED tools to download
+ - `bedtools_version`: (default: `2.28.0`) the version of BED tools to download
     and `make`
  - `galaxy_custom_tools_dir`: System path where Trackster utility tools will be
-    installed. This path should be on the `$PATH`.
+    installed. This path should be on the galaxy `$PATH` (default: `/home/galaxy/galaxy/.venv/bin`)
```
Comment (afgane) [inline README.md:18]: Related to above, if changed.

File: README.md [hunk 4]
```diff
@@ -3,24 +3,19 @@ This role is used to setup [Trackster][trackster] genome browser in Galaxy.
...
-If using this role in the context of [Galaxy CloudMan][cm], note that some of these
-variables need to match equally named ones from the [`ansible-cloudman`][acm]
-role.
 
  - `galaxy_manage_trackster`: (default: `yes`) if set, the role will run
  - `galaxy_user_name`: (default: `galaxy`) system username used for Galaxy
- - `galaxyFS_base_dir`: (default: `/mnt/galaxy`) the base path under which the
+ - `galaxy_server_dir`: (default: `/home/galaxy/galaxy`) the base path under which the
     galaxy file system is planned to be placed
- - `len_file_path`: (default: `"{{ galaxyFS_base_dir }}/galaxy-app/config/len"`)
```
Comment (afgane) [inline README.md:18]: Any particular reason not to include this var in the readme?

**Review by afgane — APPROVED** [review]
Thanks for taking this on. I made a few minor comments. Let me know what you think.

File: README.md (hunk 1 above)
Comment (drosofff) [reply README.md:5]: OK. Stricto sensu, I did tested the playbook in 18.04 ;-)

File: README.md (hunk 2 above)
Comment (drosofff) [reply README.md:13]: Well, the rational is to have a galaxy server directory (the one that is cloned with git) installed in the home directory of the galaxy user. But I am open to any arbitrary convention, except that the `shed_tools` dir is outside of the galaxy server dir in my old habit. Tell me I can change in whatever, this is a default.

File: README.md (hunk 3 above)
Comment (drosofff) [reply README.md:18]: Yes. Again just question of convention. Yet, I find the .env/bin a reasonable place for the trackster binaries

File: README.md (hunk 4 above)
Comment (drosofff) [reply README.md:18]: Yes, tiredness ;-)

**Comment by drosofff** [comment]
@afgane this is weird, I pushed a 9th commit to the ARTbio:artbio branch of my fork but it does not show up here !

**Comment by afgane** [comment]
I don't see why it wouldn’t show up so I'll just merge this and if please make a separate PR then rather than trying to chase this down.

Thanks for addressing it.

**Pull request merged**